src/App.js

```diff
@@ -1,19 +1,41 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import {Route} from 'react-router-dom';
+
+import {} from './styles/global';
+import AppContainer from './styles/containers/App';
+import CenterContenteDiv from './styles/containers/CenterContenteDiv';
+import MainMenuContainer from './styles/containers/MainMenuContainer';
+import NavLink from './components/NavLink';
+import HomeView from './components/HomeView';
+import FavoritesView from './components/FavoritesView';
+import DetailsView from './components/DetailsView';
+import {HOME_PATH, FAVORITES_PATH, DETAILS_PATH, DETAILS_VARIABLE_PATH}
+  from './config/routerConstants';
+
+export const DetailsRouterPipe = ({match}) => (
+  <div>
+    <Route path={`${match.url}/:${DETAILS_VARIABLE_PATH}`} component={DetailsView}/>
+  </div>
+);
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
+      <AppContainer>
+        <header>
+          <MainMenuContainer>
+            <CenterContenteDiv padding="15px">
```

Comment:
the plus of styled component is to create meaningful components, but not to enforce the `tagType`